.github/workflows/build.yml

```diff
@@ -67,9 +67,8 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install --upgrade \
-                      --requirement requirements-test.txt
-      - name: Run linters on all files
+          pip install --upgrade -r requirements-test.txt
```

Comment:
Prefer the long version of the flag in scripts, so it's clearer what is being done.  In other words, you should use `--requirement` here instead of `-r`.

Reply:
Ah, that's pulled in from `skeleton-generic` I'll backport an update there so it's used all the way down.

Reply:
This should be resolved in 0a2036d